app/jobs/search_keyword_job.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class SearchKeywordJob < ApplicationJob
+  def perform(keyword_id)
+    keyword = Keyword.includes(:source).where(keywords: { id: keyword_id }).first
```

Comment:
```suggestion
    keyword = Keyword.includes(:source).find_by(keywords: { id: keyword_id })
```

And please try, maybe a simple `find` would also work in that case:

```
    keyword = Keyword.includes(:source).find(keyword_id)
```